lib/iris/tests/integration/test_FieldsFileVariant.py

```diff
@@ -341,9 +341,9 @@ def test_save_packed_as_unpacked(self):
                          if hasattr(fld, 'lbpack')]
             for i_field, (new_size, old_size) in enumerate(zip(new_sizes,
                                                                old_sizes)):
-                msg = 'unpacked LBNREC({}) is <= packed({})'
-                self.assertGreater(new_size, old_size,
-                                   msg=msg.format(new_size, old_size))
+                msg = 'unpacked LBNREC({}) is < packed({})'
```

Comment:
From older PR : https://github.com/pp-mo/iris/pull/15#discussion_r29036594
> @pelson Quoi? I don't understand... could you provide some context.

Attempt to check that packed payload is smaller, as it should be.
This is admittedly weak.  I will try using LBLREC instead, in which case it should be >, not >= ...